### ISSUE TITLE
[Bug]: [perf-nightly] Milvus build diskann index , after load rasie an error"failed to load segment: follower 3 failed to load segment"

### ISSUE THREAD

**Issue by jingkl** [milvus-io/milvus#25508]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:2.2.0-20230711-66ea847c
- Deployment mode(standalone or cluster):cluster
- MQ type(rocksmq, pulsar or kafka):    pulsar
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

release_name_prefix
perf-search-1689121800
deploy_config
fouramf-server-cluster-8c16m
case_params
fouramf-client-glove1m-concurrent-diskann
other_params
 -s --deploy_mode=cluster --update_helm_file 
case_name
test_concurrent_locust_custom_parameters

server

```
perf-search-16821800-3-56-3879-etcd-0                             1/1     Running           0               52m     10.104.4.50     4am-node11   <none>           <none>
perf-search-16821800-3-56-3879-etcd-1                             1/1     Running           0               52m     10.104.1.156    4am-node10   <none>           <none>
perf-search-16821800-3-56-3879-etcd-2                             1/1     Running           0               52m     10.104.6.189    4am-node13   <none>           <none>
perf-search-16821800-3-56-3879-milvus-datacoord-6c9dfc5ddd9m4w7   1/1     Running           0               52m     10.104.19.252   4am-node28   <none>           <none>
perf-search-16821800-3-56-3879-milvus-datanode-84589c6bd6-dfd6p   1/1     Running           2 (45m ago)     52m     10.104.16.132   4am-node21   <none>           <none>
perf-search-16821800-3-56-3879-milvus-indexcoord-7cd55f745nlbjf   1/1     Running           2 (45m ago)     52m     10.104.16.129   4am-node21   <none>           <none>
perf-search-16821800-3-56-3879-milvus-indexnode-db8f9677c-zsclv   1/1     Running           0               52m     10.104.22.17    4am-node26   <none>           <none>
perf-search-16821800-3-56-3879-milvus-proxy-84dfd98d4-6twft       1/1     Running           2 (45m ago)     52m     10.104.16.131   4am-node21   <none>           <none>
perf-search-16821800-3-56-3879-milvus-querycoord-68ff8f8dczbvzl   1/1     Running           2 (44m ago)     52m     10.104.19.253   4am-node28   <none>           <none>
perf-search-16821800-3-56-3879-milvus-querynode-548b8df4cf7vpst   1/1     Running           0               52m     10.104.15.24    4am-node20   <none>           <none>
perf-search-16821800-3-56-3879-milvus-rootcoord-6597f5d8c77npxb   1/1     Running           2 (45m ago)     52m     10.104.16.130   4am-node21   <none>           <none>
perf-search-16821800-3-56-3879-minio-0                            1/1     Running           0               52m     10.104.6.190    4am-node13   <none>           <none>
perf-search-16821800-3-56-3879-minio-1                            1/1     Running           0               52m     10.104.9.218    4am-node14   <none>           <none>
perf-search-16821800-3-56-3879-minio-2                            1/1     Running           0               52m     10.104.1.158    4am-node10   <none>           <none>
perf-search-16821800-3-56-3879-minio-3                            1/1     Running           0               52m     10.104.5.25     4am-node12   <none>           <none>
perf-search-16821800-3-56-3879-pulsar-bookie-0                    1/1     Running           0               52m     10.104.6.186    4am-node13   <none>           <none>
perf-search-16821800-3-56-3879-pulsar-bookie-1                    1/1     Running           0               52m     10.104.9.216    4am-node14   <none>           <none>
perf-search-16821800-3-56-3879-pulsar-bookie-2                    1/1     Running           0               52m     10.104.4.53     4am-node11   <none>           <none>
perf-search-16821800-3-56-3879-pulsar-bookie-init-kq7zv           0/1     Completed         0               52m     10.104.4.22     4am-node11   <none>           <none>
perf-search-16821800-3-56-3879-pulsar-broker-0                    1/1     Running           0               52m     10.104.1.138    4am-node10   <none>           <none>
perf-search-16821800-3-56-3879-pulsar-proxy-0                     1/1     Running           0               52m     10.104.4.28     4am-node11   <none>           <none>
perf-search-16821800-3-56-3879-pulsar-pulsar-init-2hg57           0/1     Completed         0               52m     10.104.4.21     4am-node11   <none>           <none>
perf-search-16821800-3-56-3879-pulsar-recovery-0                  1/1     Running           0               52m     10.104.4.26     4am-node11   <none>           <none>
perf-search-16821800-3-56-3879-pulsar-zookeeper-0                 1/1     Running           0               52m     10.104.4.48     4am-node11   <none>           <none>
perf-search-16821800-3-56-3879-pulsar-zookeeper-1                 1/1     Running           0               51m     10.104.1.227    4am-node10   <none>           <none>
perf-search-16821800-3-56-3879-pulsar-zookeeper-2                 1/1     Running           0               45m     10.104.9.9      4am-node14   <none>           <none>
```

client log:
```
[2023-07-12 00:53:40,082 -  INFO - fouram]: [Base] Start build index of DISKANN for collection fouram_Um3dHQWv, params:{'index_type': 'DISKANN', 'metric_type': 'IP', 'params': {}} (base.py:486)
[2023-07-12 01:01:57,772 -  INFO - fouram]: [Time] Index run in 497.6894s (api_request.py:45)
[2023-07-12 01:01:57,772 -  INFO - fouram]: [CommonCases] RT of build index DISKANN: 497.6894s (common_cases.py:107)
[2023-07-12 01:01:57,780 -  INFO - fouram]: [Base] Params of index: [{'float_vector': {'index_type': 'DISKANN', 'metric_type': 'IP', 'params': {}}}] (base.py:500)
[2023-07-12 01:01:57,780 -  INFO - fouram]: [CommonCases] Prepare index DISKANN done. (common_cases.py:110)
[2023-07-12 01:01:57,780 -  INFO - fouram]: [CommonCases] No scalars need to be indexed. (common_cases.py:118)
[2023-07-12 01:01:57,782 -  INFO - fouram]: [Base] Number of vectors in the collection(fouram_Um3dHQWv): 1000000 (base.py:527)
[2023-07-12 01:01:57,782 -  INFO - fouram]: [Base] Start load collection fouram_Um3dHQWv,replica_number:1,kwargs:{} (base.py:283)
[2023-07-12 01:32:00,162 - ERROR - fouram]: RPC error: [get_loading_progress], <MilvusException: (code=1, message=failed to load segment: follower 3 failed to load segment, reason [UnexpectedError] Assert "ok" at /go/src/github.com/milvus-io/milvus/internal/core/src/index/VectorDiskIndex.cpp:71
 => load disk index failed)>, <Time:{'RPC start': '2023-07-12 01:32:00.159018', 'RPC error': '2023-07-12 01:32:00.162282'}> (decorators.py:108)
[2023-07-12 01:32:00,163 - ERROR - fouram]: RPC error: [wait_for_loading_collection], <MilvusException: (code=1, message=failed to load segment: follower 3 failed to load segment, reason [UnexpectedError] Assert "ok" at /go/src/github.com/milvus-io/milvus/internal/core/src/index/VectorDiskIndex.cpp:71
 => load disk index failed)>, <Time:{'RPC start': '2023-07-12 01:01:57.799079', 'RPC error': '2023-07-12 01:32:00.163296'}> (decorators.py:108)
[2023-07-12 01:32:00,163 - ERROR - fouram]: RPC error: [load_collection], <MilvusException: (code=1, message=failed to load segment: follower 3 failed to load segment, reason [UnexpectedError] Assert "ok" at /go/src/github.com/milvus-io/milvus/internal/core/src/index/VectorDiskIndex.cpp:71
 => load disk index failed)>, <Time:{'RPC start': '2023-07-12 01:01:57.782869', 'RPC error': '2023-07-12 01:32:00.163454'}> (decorators.py:108)
[2023-07-12 01:32:00,164 - ERROR - fouram]: (api_response) : <MilvusException: (code=1, message=failed to load segment: follower 3 failed to load segment, reason [UnexpectedError] Assert "ok" at /go/src/github.com/milvus-io/milvus/internal/core/src/index/VectorDiskIndex.cpp:71
 => load disk index failed)> (api_request.py:53)
[2023-07-12 01:32:00,164 - ERROR - fouram]: [CheckFunc] load request check failed, response:<MilvusException: (code=1, message=failed to load segment: follower 3 failed to load segment, reason [UnexpectedError] Assert "ok" at /go/src/github.com/milvus-io/milvus/internal/core/src/index/VectorDiskIndex.cpp:71
```

### Steps To Reproduce

```markdown
1. create a collection 
        2. build index on vector column
        3. insert a certain number of vectors
        4. flush collection
        5. build index on vector column with the same parameters
        7. count the total number of rows
        8. load collection
        9. perform concurrent operations
```

### Anything else?

[2023-07-12 00:49:54,258 -  INFO - fouram]: [check_params] scene_concurrent_locust required params: {'dataset_params': {'metric_type': 'IP', 'dim': 200, 'dataset_name': 'glove', 'dataset_size': '1m', 'ni_per': 1000}, 'collection_params': {'other_fields': []}, 'load_params': {}, 'query_params': {}, 'search_params': {}, 'index_params': {'index_type': 'DISKANN', 'index_param': {}}, 'concurrent_params': {'concurrent_number': [1, 20], 'during_time': 3600, 'interval': 20}, 'concurrent_tasks': [{'type': 'search', 'weight': 1, 'params': {'nq': 1, 'top_k': 1, 'search_param': {'search_list': 30}, 'random_data': True}}]} (params_check.py:31)

**Comment by yah01** [comment]
same as #25350

**Comment by yah01** [comment]
/assign @jingkl 
after increasing the system conf, this should have been fixed

**Comment by jingkl** [comment]
This problem has been fixed， fix image：master-20230719-e418ab2f
